app/services/retrieve_position_data.rb

```diff
@@ -47,10 +47,15 @@ def query_format
         OPTIONAL { ?position pq:P768 ?district . }
         OPTIONAL { ?position pqv:P580 [wikibase:timeValue ?position_start; wikibase:timePrecision ?position_start_precision] . }
         OPTIONAL { ?merged_then_deleted owl:sameAs ?person }
+        BIND(ceil((year(?term_start) - year(?position_start)) * 365.2425 +
+                  (month(?term_start) - month(?position_start)) * 30.4375 +
+                  (day(?term_start) - day(?position_start))) AS ?days_between)
         FILTER (!bound(?term) || ?term = ?page_term)
         FILTER (
           !bound(?term_start) || !bound(?position_start) ||
-          ?term_start <= ?position_start
+          (?position_start_precision = 9 && year(?term_start) = year(?position_start)) ||
+          (?position_start_precision = 10 && year(?term_start) = year(?position_start) && month(?term_start) = month(?position_start)) ||
```

Comment:
Similarly, I guess this should be `year of term before year of position OR years the same and month is earlier`

Reply:
Changed to:
`year of term before year of position OR years the same and month is *equal or* earlier` which I think you meant